src/Styleguide/Pages/Artwork/index.js

```diff
@@ -48,25 +46,12 @@ export class Artwork extends Component {
           <Col sm={8}>
             <ArtworkSlider>
               <ImageCarousel src={slideshowImages} />
-              <ArtworkImage>TODO: Artwork Image</ArtworkImage>
-              <SlideIndicatorDots>. . .</SlideIndicatorDots>
-              <UtilityButtons>
-                <FavoriteButton>TODO: Heart Icon</FavoriteButton>
-                <ShareButton>TODO: Share Icon</ShareButton>
-              </UtilityButtons>
```

Comment:
Oh, merge conflict in the making! I’ll undo my similar changes locally.

Reply:
Sorry - I should not have touched those but figured you forgot to remove them @alloy. We trying to be helpful:)

Reply:
It’s all good! 👌